cli/src/build/onboarding/ui/steps/android-sa-gcp.tsx

```
@@ -0,0 +1,574 @@
+// only after the parent measured it fits). Dense: the Alert / blank lines are
+// dropped — the failure is conveyed by a single red ✖-style line above the
+// choices — and the Select keeps its window capped so a long message can't push
+// the control off-screen.
+
+export interface SaJsonValidationFailedStepProps {
+  message: string
+  onChoose: (choice: 'retry' | 'save-anyway' | 'oauth') => void
+  dense?: boolean
+}
+
+export const SaJsonValidationFailedStep: FC<SaJsonValidationFailedStepProps> = ({ message, onChoose, dense = false }) => {
```

ESLint

[error] 'dense' is assigned a value but never used. Allowed unused args must match /^_/u.

(unused-imports/no-unused-vars)